server/config/config.go

```diff
@@ -605,6 +695,10 @@ type ScheduleConfig struct {
 	StoreBalanceRate float64 `toml:"store-balance-rate" json:"store-balance-rate,omitempty"`
 	// StoreLimit is the limit of scheduling for stores.
 	StoreLimit map[uint64]StoreLimitConfig `toml:"store-limit" json:"store-limit"`
+	// StoreRuleList is the history of store rules.
+	StoreRuleList StoreRules `toml:"store-rule-list" json:"store-rule-list"`
+	// StoreRuleLimit is the limit of scheduling for store rules.
+	StoreRuleLimit map[string]StoreLimitConfig `toml:"store-rule-limit" json:"store-rule-limit"`
```

Comment:
How do we distinguish `StoreRuleLimit` and `StoreLimit`? I think it is a bit complicated.

Reply:
Yes. If it is not exposed to users, then where to save the map(which is used when a new store comes)

Reply:
Something like `InternalConfig`, if other situations like this will happen in the future?